src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -1133,6 +1133,12 @@ inline intx byte_size(void* from, void* to) {
   return (address)to - (address)from;
 }
 
+// Find log2 value greater than this input
```

Comment:
This should be in utilities/powerOfTwo.hpp and then as general method.

Reply:
Done.  Yes that's a better place.